backend/database/actionsWithTables/producerActions.js

```diff
@@ -0,0 +1,32 @@
+const { Producer } = require('../tables/producer');
+
+const addNewProducer = async (data) => {
+    return await Producer.create({
+        name: data.name,
+    });
+};
+
+const returnProducer = async () => {
```

Comment:
It is better to name function in correct noun for action
```suggestion
const returnAllProducers = async () => {
```
or
```suggestion
const getAllProducers = async () => {
```